kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -319,15 +319,21 @@ def _trim_job_tag(self, name):
     # Labels attached: namespace, pod
     # As a message the phase=Pending|Running|Succeeded|Failed|Unknown
     # From the phase the check will update its status
+    # Also submits as a gauge metric with tags so it is visualisable over time
     def kube_pod_status_phase(self, message, **kwargs):
         """ Phase a pod is in. """
+        metric_name = self.NAMESPACE + '.pod.status_phase'
         # Will submit a service check which status is given by its phase.
         # More details about the phase in the message of the check.
         check_basename = self.NAMESPACE + '.pod.phase'
         for metric in message.metric:
             self._condition_to_tag_check(metric, check_basename, self.pod_phase_to_status,
                                          tags=[self._label_to_tag("pod", metric.label),
                                                self._label_to_tag("namespace", metric.label)] + self.custom_tags)
+            # More verbose tagging for gauge vs service check
+            tags = [self._format_tag(label.name, label.value) for label in metric.label] + self.custom_tags
```

Comment:
Which additional tags are you looking to get?

Reply:
The only ones I actually need are "pod", "namespace" and "phase". Happy to trim it down to just those. I added it like this to mirror the functionality of adding it to the whitelist of tags (which I believe would just add all tags from the metric?).

Reply:
It might be helpful to just trim it down to those + the custom tags as that'll make it easier to maintain 👍